op-deployer/isthmus-predeploys.json

```diff
@@ -0,0 +1,5 @@
+{
+    "L1Block": "0xff256497d61dcd71a9e9ff43967c13fde1f72d12",
+    "GasPriceOracle": "0x93e57a196454cb919193fa9946f14943cf733845",
+    "OperatorFeeVault":"0x4fa2be8cd41504037f1838bce3bcc93bc68ff537"
+}
```

Comment:
We shouldn't commit this, just sharing for operational reasons. Can be checked against https://specs.optimism.io/protocol/isthmus/derivation.html#l1block-deployment